src/rules/trailing-spaces.ts

```diff
@@ -52,11 +52,11 @@ export default class TrailingSpaces extends RuleBuilder<TrailingSpacesOptions> {
         description: 'With `Two Space Linebreak = true`',
         before: dedent`
           # H1
-          Line with trailing spaces and tabs.  
+          Line with trailing spaces and tabs.  ${''}
         `,
         after: dedent`
           # H1
-          Line with trailing spaces and tabs.  
+          Line with trailing spaces and tabs.  ${''}
```

Comment:
It looks like there should be a tab present here based on the text. But maybe that is incorrect.